### PR TITLE
updated german translation

### PR DESCRIPTION
**Pull request by SethosII**
@wookietreiber can you take a look at this?

### CODE REVIEW

File: messageDir/de.lua [hunk 1]
```diff
@@ -100,205 +136,252 @@ Alternativ können Sie die Umgebungsvariable LMOD_DISABLE_SAME_NAME_AUTOSWAP auf
...
+     pin_hlp   = "Beim Wiederherstellen von Modulen die angegebene Version nutzen und die Standard-Version ignorieren.",
+     avail_hlp = "Zeigt Standard-Module nur in Verbindgung mit \"avail\".",
+     quiet_hlp = "Zeigt keine Warnungen an.",
+     exprt_hlp = "Experten Modus.",
+     terse_hlp = "Aktiviert die maschinenlesbare Ausgabe für die Befehle: \"list\", \"avail\", \"spider\", \"savelist\".",
+     initL_hlp = "Lädt Lmod zum ersten mal in eine Nutzer-Shell.",
+     latest_H  = "Lädt die aktuellste Version eines Moduls ohne Beachtung von Standard-Modulen.",
+     cache_hlp = "Behandelt die Cache-Dateien als veraltet.",
+     novice_H  = "Schaltet die Experten- und Stille-Schalter ab.",
+     raw_hlp   = "Gibt Modul-Dateien im Rohformat aus, wenn wenn show verwendet wird.",
```
Comment (wookietreiber) [inline messageDir/de.lua]: doppeltes **wenn**

**Comment by rtmclay** [comment]
Many thanks for the updated "de" translations.  I hope that Lmod's german users appreciate it!

File: messageDir/de.lua (hunk 1 above)
Comment (SethosII) [inline messageDir/de.lua]: Oh, Danke! Gibt es für sowas ein Regex? ^^"

**Comment by SethosII** [comment]
No problem. The change suggested by @wookietreiber should be included. This line also has an unquoted command. See #226.

**Pull request merged**